src/Symfony/Component/Process/Tests/ExecutableFinderTest.php

```diff
@@ -98,6 +98,7 @@ public function testFindWithOpenBaseDir()
             $this->markTestSkipped('Cannot test when open_basedir is set');
         }
 
+        putenv('PATH='.\dirname(\PHP_BINARY));
```

Comment:
Shouldn't be restored to previous value at end of the test?

Reply:
Already the case, see https://github.com/symfony/symfony/pull/51297/files#diff-da8254fae04a586ecebf906fa89d9c726042004da48af989f9b68aeb98ad6776L24